### DISCUSSION TITLE
Update to version 0.71.0 or above

### DISCUSSION THREAD

**Discussion by lnwanghao** [react-native-tvos/react-native-tvos#463] [General Discussions]
Now the latest version is 0.69.6 about react native tvos.
Is there any plan to update RN to version 0.71 recently?

RN lower 0.71 is too slow when build release and migration is more difficult.

**Comment by huanghzn** [comment]
Interested to know about the ETA for 0.71 too, and 0.72 is soon to be released.
I had a project with expo SDK 48 and really wanted to get it working with react-native-tvos fork to support android tv.

edit: typo

**Comment by douglowder** [comment]
The 0.71 upgrade is now in active development. Changes merged from upstream 0.71. Initial fixes implemented to get tvOS working, now will make sure Android works.

**Reply by Poheart** [reply]
Thank you for the update!

**Comment by douglowder** [comment]
First prerelease is available for testing, see https://github.com/react-native-tvos/react-native-tvos/discussions/495

**Comment by douglowder** [comment]
0.71 is out, we can close this
